deploy/12_old_catalysts/05_set_catalysts_minter_and_ admin.ts

```diff
@@ -0,0 +1,72 @@
+import {DeployFunction} from 'hardhat-deploy/types';
```

Comment:
maybe worth deleting the trailing space in file name

Reply:
done.